app/helpers/application_helper.rb

```diff
@@ -46,6 +46,28 @@ def bigbluebutton_endpoint_default?
     Rails.configuration.bigbluebutton_endpoint_default == Rails.configuration.bigbluebutton_endpoint
   end
 
+  def bbb_endpoint
```

Comment:
This new code in the helper is duplicated in the controller. You can either keep it in the helper and include the helper in the controller when required OR, keep it in the controller and declare each method as a helper.